src/main/java/org/jabref/gui/groups/GroupSelector.java

```diff
@@ -407,13 +403,13 @@ private void showPopup(MouseEvent e) {
             if (node.canBeEdited()) {
                 editGroupPopupAction.setEnabled(false);
                 addGroupPopupAction.setEnabled(false);
-                removeGroupAndSubgroupsPopupAction.setEnabled(false);
+                //removeGroupAndSubgroupsPopupAction.setEnabled(false);
```

Comment:
The new button is always enabled which is not what you want. Thus this old code functions as a reminder for myself to change it.